info.json

```diff
@@ -41,10 +41,17 @@
         ],
         "globalVariables": [],
         "connectors": [
+            {
+                "name": "Active Directory",
+                "apiName": "activedirectory"
+            },
+            {
+                "name": "Fortinet FortiEDR",
```

Comment:
Why we added these two connectors?

Reply:
How these connectors went missing?

Reply:
Actually, these two connectors were installed through SF SP and this solution pack is dependent on SF SP that's why I was not added that, but I am thinking if by mistake one of the connectors will uninstall by any analyst then this SP use case will fail. Hence for the safe side, I have added these connectors

If the connectors are already installed then it will ignore otherwise install using this SP.

Reply:
Got it.